src/app/modules/data-set/data-set-instance/data-set-instance.component.ts

```diff
@@ -15,9 +19,11 @@ import { AnnotationStatus, InstanceType } from '../model/enums/enums.model';
 export class DataSetInstanceComponent implements OnInit {
 
   @Input() public instances: DataSetInstance[] = [];
-  @Input() public instancesType: string = '';
+  @Input() public filter: InstanceFilter = InstanceFilter.All;
+  public instanceFilter = InstanceFilter;
```

Comment:
What is the difference between these two filters?

Reply:
First one,  _filter_, is enum value used in **.ts** to filter instances.
Secondly, _instanceFilter_, is enum type which is used in **.html** in **ngFor**.